session/src/main/java/org/apache/iotdb/session/SessionUtils.java

```diff
@@ -149,4 +158,32 @@ private static void getValueBufferOfDataType(
             String.format("Data type %s is not supported.", dataType));
     }
   }
+
+  public static List<EndPoint> parseSeedNodeUrls(List<String> nodeUrls) {
+    if (nodeUrls == null) {
+      return Collections.emptyList();
+    }
+    List<EndPoint> endPointsList = new ArrayList<>();
+    for (String nodeUrl : nodeUrls) {
+      EndPoint endPoint = parseNodeUrl(nodeUrl);
+      endPointsList.add(endPoint);
```

Comment:
`parseNodeUrl`  method may return `null`, we can not add one `null` endpoint to the `endPointsList`